motorlib/units.py

```diff
@@ -8,12 +8,12 @@
 
 ('m/s', 'ft/s', 3.28),
 
-('n', 'lbf', 0.2248),
+('N', 'lbf', 0.2248),
 
-('ns', 'lbfs', 0.2248),
+('Ps', 'lbfs', 0.2248),
```

Comment:
```suggestion
('Ns', 'lbfs', 0.2248),
```

Reply:
Other than this typo it looks good. Thanks!

Reply:
Thanks, came across data and calculations for grains on http://www.nakka-rocketry.net. My plan was to put that in a Python program. Just as I was about to begin I quickly checked git and came across your code. This is what I was thinking so I might be making more contributions in the future.